tests/basic/async_dr_test.go

```
@@ -817,10 +985,252 @@ func hardSetConfig(configPath string) error {
 
 func waitForPodsToBeRunning(context *scheduler.Context, expectedFail bool) {
 	log.Infof("Verifying Context [%v]", context.App.Key)
-	err := Inst().S.WaitForRunning(context, 5 * time.Minute, 10 * time.Second)
+	err := Inst().S.WaitForRunning(context, 5*time.Minute, 10*time.Second)
 	if expectedFail {
 		log.FailOnNoError(err, "Pods are up on destination, they shouldn't be up")
 	} else {
 		log.FailOnError(err, "Error waiting for pods to be up")
 	}
-}
\ No newline at end of file
+}
+
+func validateOperatorMigFailover(namespace, clusterType, opName, crName string, podCount int, clusterwide bool) {
+	cpName := defaultClusterPairName + time.Now().Format("15h03m05s")
+	kubeConfigPathSrc, err := GetCustomClusterConfigPath(asyncdr.FirstCluster)
+	log.FailOnError(err, "Failed to get source configPath: %v", err)
+	kubeConfigPathDest, err := GetCustomClusterConfigPath(asyncdr.SecondCluster)
+	log.FailOnError(err, "Failed to get destination configPath: %v", err)
+	if clusterType == "asyncdr" {
+		err = ScheduleBidirectionalClusterPair(cpName, namespace, "", storkapi.BackupLocationType(defaultBackupLocation), defaultSecret, "async-dr", asyncdr.FirstCluster, asyncdr.SecondCluster)
+	} else {
+		err = ScheduleBidirectionalClusterPair(cpName, namespace, "", "", "", "sync-dr", asyncdr.FirstCluster, asyncdr.SecondCluster)
+	}
+	log.FailOnError(err, "Failed creating bidirectional cluster pair")
+	log.Infof("Start migration schedule and perform failover")
+	migrationSchedName := migrationSchedKey + time.Now().Format("15h03m05s")
+	extraArgs := map[string]string{
+		"namespaces":             namespace,
+		"kubeconfig":             kubeConfigPathSrc,
+		"exclude-resource-types": "ClusterServiceVersion,operatorconditions,OperatorGroup,InstallPlan,Subscription",
+		"exclude-selectors":      "olm.managed=true",
+	}
+	extraArgsFailoverFailback := map[string]string{
+		"kubeconfig": kubeConfigPathDest,
+	}
+	err = patchStashStrategy(crName)
```

I think the err is not handled.

---

Done